### PR TITLE
main/rhash: upgrade to 1.3.6

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

**Comment by jirutka** [comment]
There are many missing binaries:

```diff
-etc/
-etc/rhashrc
 usr/
 usr/bin/
-usr/bin/ed2k-link
-usr/bin/gost-hash
-usr/bin/has160-hash
-usr/bin/magnet-link
 usr/bin/rhash
-usr/bin/sfv-hash
-usr/bin/tiger-hash
-usr/bin/tth-hash
-usr/bin/whirlpool-hash
 usr/lib/
```

**Comment by prspkt** [comment]
Thanks! It should be good now.

File: main/rhash/APKBUILD [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
 arch="all"
 license="custom"
+makedepends="libressl-dev"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-libs"
-source="$pkgname-$pkgver.tar.gz::http://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver-src.tar.gz"
+source="http://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver-src.tar.gz"
 builddir="$srcdir/RHash-$pkgver"
 
+prepare() {
+	# include endian.h on musl
```
Comment (jirutka) [inline main/rhash/APKBUILD]: Please always call `default_prepare` in custom `prepare()` function.

Comment (prspkt) [reply main/rhash/APKBUILD]: @jirutka Noted and fixed.
Thanks.

**Comment by ncopa** [comment]
fixed with e9683d537d744d14ad7dfde6f1b39370705e831f and cd0f7dce03ab95b48d00da69bdc9a0d29f86832f

**Pull request closed without merging**